docs/src/main/asciidoc/security-basic-auth-concept.adoc

```diff
@@ -0,0 +1,36 @@
+[id="security-basic-auth-concept"]
+= Basic Authentication
+ 
+HTTP Basic Authentication uses fields in the HTTP header and is the easiest to set up. Also, it is one of the least resource-demanding techniques that enforce access controls to the Web resources without requiring HTTP cookies, session identifiers, or login pages.
+
+In the context of an HTTP request, Basic authentication is a method for an HTTP user agent, such as a web browser, to provide a user name and password when creating a request. In Basic HTTP Authentication, a request contains a header field in the form of `Authorization: Basic <credentials>`, where credentials are the Base64 encoding of a user ID and password joined by a colon as described in the following example.
+
+====
+.Example
+ 
+If the user name is `Alice` and the password is `secret`, the HTTP authorization header looks authorization as `Authorization: Basic  QWxjZTpzZWNyZXQ=`, where `QWxjZTpzZWNyZXQ=` is a Base64 encoded representation of the `Alice:secret` string.
+====
+ 
+The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
```

Comment:
```suggestion
The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. 
The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. 
Basic Authentication should be used with HTTPS to provide confidentiality.
```

One line per sentence. People can use Basic Auth without HTTPS, something that is mentioned again in the section below.

Perhaps this paragraph should be combined with the first section of the second? Not sure.

Reply:
Why we cant write 2 sentences per line, if the context is the same and we dont want to create a lot of white space?

Reply:
It doesn't leave whitespace or create a break in the paragraph. 
It does help with diffs over time when reviewing edits.

https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/doc-reference.adoc#semantic-line-breaks

https://rhodesmill.org/brandon/2012/one-sentence-per-line/

Reply:
Changed to one sentence per line.